crypto/pem/pem_lib.c

```diff
@@ -661,177 +694,288 @@ int PEM_read(FILE *fp, char **name, char **header, unsigned char **data,
 }
 #endif
 
-int PEM_read_bio(BIO *bp, char **name, char **header, unsigned char **data,
-                 long *len)
+/* Some helpers for PEM_read_bio_flags(). */
+
+/* TODO Presumably this should be a table lookup, but are these the right ranges? */
+static int isb64(char c)
```

Comment:
I wasn't super keen on exposing that stuff to the whole library and leaving it in encode.c, nor was I super enthusiastic about splitting it out into a more generic place.  But, if that's how it has to be, I can do it...